product-landing-page.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>product-landing-page</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+<body>
+  <div id="page-wrapper">
+    <header id="header">
+      <div class="logo">
+        <img id="header-img" src="access/1j+ojFVDOMkX9Wytexe43D6kh...GGpRZImRzLwXs1M3EMoAJtlyErgfZt9fwz" alt="original trombones logo"/>
+      </div>
+      <nav id="nav-bar">
+        <ul class="">
+          <li class="color"><a class="nav-link" href="#home">Home</a></li>
+          <li class="color"><a class="nav-link" href="#how-it-works">How It Works</a></li>
+          <li class="color"><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
```

Comment:
- Your #nav-bar should always be at the top of the viewport.

Reply:
ok sir